include/kframework/foundry.md

```diff
@@ -1076,12 +1076,12 @@ Utils
 - `#setCode ACCTID CODE` sets the code of a given account.
 
 ```k
-    syntax KItem ::= "#setCode" Int ByteArray [klabel(foundry_setCode)]
- // -------------------------------------------------------------------
+    syntax KItem ::= "#setCode" Int Bytes [klabel(foundry_setCode)]
+ // ---------------------------------------------------------------
     rule <k> #setCode ACCTID CODE => . ... </k>
          <account>
            <acctID> ACCTID </acctID>
-           <code> _ => #if #asWord(CODE) ==Int 0 #then .ByteArray:AccountCode #else {CODE}:>AccountCode #fi </code>
+           <code> _ => #if #asWord(CODE) ==Int 0 #then .Bytes:AccountCode #else {CODE}:>AccountCode #fi </code>
            ...
          </account>
 ```
```

Comment:
Do we actually need this semantic kast anymore with simplified types? Can we remove the `:AccountCode` and the `{ ... }:>AccountCode` and it still builds OK?

Reply:
It builds without an error now.